tests/client/disco/containers/TestInstallButton.js

```diff
@@ -1,18 +1,9 @@
 import React from 'react';
-import {
-  Simulate,
-  findRenderedComponentWithType,
-  renderIntoDocument,
-} from 'react-addons-test-utils';
+import { Simulate, renderIntoDocument } from 'react-addons-test-utils';
 import { findDOMNode } from 'react-dom';
-import { Provider } from 'react-redux';
-import { createStore } from 'redux';
-
-import StateulInstallButton, {
-  InstallButton,
-  mapDispatchToProps,
-  mapStateToProps,
-} from 'disco/containers/InstallButton';
+import * as addonManager from 'disco/addonManager';
```

Comment:
Is there a reason to do this instead of `import { AddonManager } from 'disco/addonManager'`.